homeassistant/components/arve/sensor.py

```diff
@@ -0,0 +1,101 @@
+"""Support for Arve devices."""
```

Comment:
```suggestion
"""Sensor platform for Arve devices."""
```
getting nitpicky (which is a good sign) 😉